challenges/objects-arrays.js

```diff
@@ -46,15 +93,26 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
+
+for(let i = 0; i < graduates.length; i++) {
+	universities.push(graduates[i].university);
+}
+
+console.log(universities.sort());
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
+///still need to get the email in there...
 const contactInfo = [];
+
+for(let i = 0; i < graduates.length; i++) {
+contactInfo.push(graduates[i].first_name);
+}
+
 console.log(contactInfo);
 
 
```

Comment:
We will go over the uni problem in your one-on-one! There is an easy method that makes it pretty easy once you know it.